templates/html.tmpl

```diff
@@ -0,0 +1,16 @@
+<h2>{{ .Heading }}</h2>
```

Comment:
An example of how GitHub renders this HTML table can be viewed here:
https://github.com/mdb/terraputs/tree/html-output#outputs